samples/listener/trigger_on_label_removed_email.bal

```diff
@@ -37,7 +37,7 @@ gmail:GmailConfiguration gmailConfig = {
 listener gmailListener:Listener gmailEventListener = new(port, gmailConfig,  project, pushEndpoint);
 
 service / on gmailEventListener {
-   remote function onLabelRemovedEmail(gmailListener:ChangedLabel changedLabeldMsg) returns error? {
-           log:printInfo("Label Removed Mail : " , changedLabeldMsg);
+   remote function onLabelRemovedEmail(gmailListener:ChangedLabel changedLabel) returns error? {
```

Comment:
`onEmailLabelRemoved`? What I get from function name here is, I get an email that got his label removed.

Reply:
Yes, It means, When a label removed from a particular email, it will trigger and give the email and removed labels.

Reply:
Renamed as we discussed in offline.